src/cli.js

```diff
@@ -20,6 +21,27 @@ const configDir = program.configDir || './.storybook';
 
 const configPath = path.resolve(`${configDir}`, 'config');
 
+const babelConfig = loadConfig(configDir);
+babelConfig.babelrc = false;
+
+require('babel-register')(babelConfig);
+require('babel-polyfill');
+
+const fileExts = ['jpg', 'png', 'gif', 'eot', 'svg', 'ttf', 'woff', 'woff2']
```

Comment:
Move the following code into a file called `loaders.js` like this: 

```js
// This is just an example:
export default {
   "jpg, png, gif": (ext, moduleName) => {
       return `/the/filename.${ext}`;
    },

    "css, scss, sass": (ext, moduleName) => null
}
```

Then create an option to load user's loaders instead of our default. 
We can add a guide in README on how to do that.